src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Store } from '@ngrx/store';
+import { Observable, Subscription } from 'rxjs';
+import { filter } from 'rxjs/operators';
+import { HearingActualsStateData } from '../../../models/hearingActualsStateData.model';
+import { HearingActualsMainModel } from '../../../models/hearingActualsMainModel';
+import { HearingResult } from '../../../models/hearings.enum';
+import { LovRefDataModel } from '../../../models/lovRefData.model';
+import { LovRefDataService } from '../../../services/lov-ref-data.service';
+import * as fromHearingStore from '../../../store';
+
+@Component({
+  selector: 'exui-hearing-stage-result',
+  templateUrl: './hearing-stage-result.component.html',
+  styleUrls: ['./hearing-stage-result.component.scss']
+})
+export class HearingStageResultComponent implements OnInit, OnDestroy {
+  public hearingStageResultForm: FormGroup;
+  public validationErrors: { id: string, message: string }[] = [];
+  public hearingResultType: string;
+  public caseTitle: string;
+  public hearingTypes$: Observable<LovRefDataModel[]>;
+  public completeHearingActualReasons$: Observable<LovRefDataModel[]>;
+  public adjournHearingActualReasons$: Observable<LovRefDataModel[]>;
+  public cancelHearingActualReasons$: Observable<LovRefDataModel[]>;
+  public hearingActuals: HearingActualsMainModel;
+  public sub: Subscription;
+
+  constructor(private readonly hearingStore: Store<fromHearingStore.State>,
+              private readonly lovRefDataService: LovRefDataService,
+              private readonly formBuilder: FormBuilder) {
+  }
+
+  public get hearingResultEnum() {
+    return HearingResult;
+  }
+
+  public ngOnInit(): void {
+    this.hearingStageResultForm = this.formBuilder.group({
+      hearingStage: [''],
+      hearingResult: ['', Validators.required],
+      adjournedReason: [''],
+      cancelledReason: ['']
+    });
+    this.hearingTypes$ = this.lovRefDataService.getListOfValues('HearingType', 'SSCS');
```

Comment:
could you remove these please in favour of using `RefDataResolver` as the serviceId is hardcoded

Reply:
that is good spot, thanks @paddybasi

Reply:
@paddybasi I have now updated the code as we discussed.